ui/lib/utils.ts

```diff
@@ -45,3 +47,9 @@ export function convertGitURLToGitProvider(uri: string) {
 export function pageTitleWithAppName(title: string, appName?: string) {
   return `${title}${appName ? ` for ${appName}` : ""}`;
 }
+
+export function statusSortHelper(suspended: boolean, conditions: Condition[]) {
```

Comment:
If you accept an object instead, you don't have to create a new function call every time:
```suggestion

interface Statusable {
  conditions: Condition[];
  suspended: boolean;
}
export function statusSortHelper({suspended, conditions}: Statusable) {
```
Then this:

```js
sortValue: ({ conditions, suspended }) =>
              statusSortHelper(suspended, conditions),
```

becomes

```js
sortValue: statusSortHelper,
```

Reply:
`statusSortHelper` wouldn't need any arguments?